frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,156 @@
+<script lang="ts">
+	export let anime: any;
+
+	import ScrollArea from '$components/shared/ScrollArea.svelte';
+	import SidebarDetails from '$components/pages/anime_info/SidebarDetails.svelte';
+
+	// icons
+	import PlayCircle from '$icons/PlayCircle.svelte';
+	import Read from '$icons/Read.svelte';
+	import Listen from '$icons/Listen.svelte';
+	import Video from '$icons/Video.svelte';
+	import Download from '$icons/Download.svelte';
+	import Share from '$icons/Share.svelte';
+	import SettingsOutline from '$icons/SettingsOutline.svelte';
+	import Edit from '$icons/Edit.svelte';
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end xl:gap-10">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="xl:w-60 2xl:w-96 xl:rounded-xl 2xl:rounded-3xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
+			<div>
+				<h2 class="font-bold 2xl:!text-5xl 3xl:!text-6xl">{anime.titles.eng}</h2>
```

Comment:
Why are there so many important class?

Reply:
Oky, will remove unuseful important classes

Reply:
> `3xl:!text-6xl`

3xl is not a valid size

Reply:
> 3xl is not a valid size
Its for hd monitors above 1920px
Then your suggestion?

Reply:
> Its for hd monitors above 1920px
> Then your suggestion?

use VW and VH